src/main/UI/PasswordGeneratorBox.java

```diff
@@ -0,0 +1,134 @@
+package UI;
+
+import controller.PasswordGeneratorUIControl;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.Objects;
+
+public class PasswordGeneratorBox implements ActionListener {
+    DialogBox dialogBox;
+    ComboBox comboBox;
+    TextArea textArea;
+    Button button;
+    Label label;
+    RadioBox lcbox, ucbox, numbox, spcbox;
+    Icon icon;
+
+
+    public void createPasswordGeneratorBox() {
+
+        // Generating a DialogBox object
+        dialogBox = new DialogBox();
+        dialogBox.createDialogBox("Password Generator", 400, 360);
+        dialogBox.jframe.setVisible(true);
+        dialogBox.jframe.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+        dialogBox.jframe.setResizable(false);
+        Color c = new Color(204,255,255);
+        dialogBox.jframe.getContentPane().setBackground(c);
+
+        // Create a label
+        label = new Label();
+        label.createLabel(90,40,150,20,dialogBox.jframe, "Password Length:", Color.blue);
+
+        //creating a Combobox to get the password length from the user
+        comboBox = new ComboBox();
+        String[] comboBoxItems = {"8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20"};
+        comboBox.createComboBox(dialogBox.jframe, comboBoxItems, Color.white,230,40,50,20);
+        comboBox.comboBox.setForeground(Color.black);
+
+        //Creating radio checkboxes to get the password type from the user
+        //Creating radiobox for Lowercase letters
+        lcbox = new RadioBox();
+        lcbox.createRadioButton(dialogBox.jframe, "Lowercase", Color.white,110,70,170,30);
```

Comment:
Please run Code->Reformat code so that the whitespace follows Java conventions. Also, is there a way to get rid of the magic numbers?